<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtDigitalAttenuator</class>
 <widget class="QWidget" name="QtDigitalAttenuator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>630</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_18">
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
       </property>
       <item row="2" column="1">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Current</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QLCDNumber" name="currentattenuation_lcdNumber">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>80</height>
                 </size>
                </property>
                <property name="smallDecimalPoint">
                 <bool>false</bool>
                </property>
                <property name="digitCount">
                 <number>6</number>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::SegmentStyle::Filled</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <layout class="QVBoxLayout" name="verticalLayout">
                <property name="sizeConstraint">
                 <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Set:</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="statusSet_label">
                    <property name="text">
                     <string>----</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="db_label">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>dB</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <widget class="QLineEdit" name="model_lineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Model:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Set</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="1" column="2">
             <layout class="QGridLayout" name="gridLayout_14">
              <item row="0" column="1">
               <widget class="QGroupBox" name="groupBox_7">
                <property name="title">
                 <string>Attenuation</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_13">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_15">
                   <item row="1" column="3" rowspan="2">
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <item>
                      <widget class="QCheckBox" name="autoset_checkBox">
                       <property name="text">
                        <string>autoset</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="set_pushButton">
                       <property name="text">
                        <string>Set</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item row="1" column="2" rowspan="2">
                    <widget class="QLabel" name="label_7">
                     <property name="text">
                      <string>dB</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1" rowspan="2">
                    <widget class="QDoubleSpinBox" name="attenuation_doubleSpinBox">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="prefix">
                      <string>-</string>
                     </property>
                     <property name="suffix">
                      <string/>
                     </property>
                     <property name="decimals">
                      <number>2</number>
                     </property>
                     <property name="maximum">
                      <double>200.000000000000000</double>
                     </property>
                     <property name="singleStep">
                      <double>0.250000000000000</double>
                     </property>
                     <property name="stepType">
                      <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="1">
        <widget class="QGroupBox" name="deviceConsole_groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QLineEdit" name="write_data">
            <property name="text">
             <string>att-11.25</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="send_pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPlainTextEdit" name="data_plainTextEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEdit">
            <property name="text">
             <string>Set: att-xxx.xx/att-xx.xx Get: READ</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="device_comboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="refreshDevices_toolbutton">
           <property name="toolTip">
            <string>Refresh</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="connect_pushButton">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="disconnect_pushButton">
           <property name="text">
            <string>Disconnect</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="deviceConsole_pushButton">
         <property name="text">
          <string>Show Device Console</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
